ui/components/GithubDeviceAuthModal.tsx

```diff
@@ -54,7 +66,20 @@ const ModalContent = styled(({ codeRes, onSuccess, onError, className }) => {
   return (
     <div className={className}>
       <Pad wide center>
-        <Text size="extraLarge">{codeRes.userCode}</Text>
+        <Text
+          size="extraLarge"
+          onClick={handleCopy}
+          style={{
+            display: "inline-flex",
+          }}
+        >
+          {codeRes.userCode}
+          <PointerIcon
```

Comment:
💡 I did this as a button in `YamlView` - if that code would work here you could get rid of the styled component with `cursor: pointer`!

Reply:
good point :sparkles: